sql-indent.el

```diff
@@ -1966,14 +1971,21 @@ This is an indentation adjuster and needs to be added to the
       ;; line, not the operator
       (cond ((looking-at sqlind-operator-regexp)
 	     (let ((ofs (length (match-string 0))))
-	       (forward-line -1)
-	       (end-of-line)
-	       (sqlind-backward-syntactic-ws)
-	       ;; Previous function leaves us on the first non-white-space
-	       ;; character.  This might be a string terminator (') char, move
-	       ;; the cursor one to the left, so 'forward-sexp' works correctly.
-	       (ignore-errors (forward-char 1))
-	       (forward-sexp -1)
+	       (goto-char
```

Comment:
I don't think it is a good idea to have such a big code block as an argument to `goto-char`.  It seems that `goto-char` is only needed inside the `if` statement, the "else" part can stay as is, without the last `(point)` call.